active/website.md

```diff
@@ -0,0 +1,62 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This working group will manage Django's official website, **djangoproject.com** on behalf of the Django Software Foundation
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Ensure that information on the website is accurate 
+
+
+### Delegated responsibilities:
+- Chair, Co-chair and Board Liaison can make final decision of  sections of the website that might not conform with DSF's CoC.
+- Members can work on bugs
+- Members can tag,label and respond to issues.
+- Member can maintain and monitor the website including updating versions
+- Mentor new contributors to the website
+- Chair, Co-Chair and Board Liaison can sign off on new features
+
+## Initial membership
+
+- Chair: TBA
+- Co-Chair: TBA
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): Thibaud Colas
+- Other members:
+    - Jacob Kaplan-Moss
```

Comment:
I’m not clear that Jacob said he was interested in joining this group. He started #2 but I think it was just an admin task for him?

```suggestion
- Other members:
```

Reply:
Indeed, I've got enough on my plate, I can't take this on as well.